packages/shortest/src/ai/tools/index.ts

```diff
@@ -30,6 +30,42 @@ export const AITools = [
       required: ["action", "username", "password"],
     },
   },
+  {
+    name: "check_email",
+    description: "Fetch, decode and render received email in browser",
+    input_schema: {
+      type: "object",
+      properties: {
+        action: {
+          type: "string",
+          enum: ["check_email"],
+          description: "Render the received email in a new tab"
+        }
+      },
+      required: ["action", "email"]
+    }
+  },
+  {
+    name: "sleep",
+    description: "Pause test execution for specified duration",
+    input_schema: {
+      type: "object",
+      properties: {
+        action: {
+          type: "string",
+          enum: ["sleep"],
+          description: "The action to perform"
+        },
+        duration: {
+          type: "number",
+          description: "Duration to sleep in milliseconds (e.g. 5000 for 5 seconds)",
+          minimum: 0,
+          maximum: 60000
+        }
+      },
+      required: ["action", "duration"]
+    }
+  },
```

Comment:
Another tool that we needed to ship with "check_email" is "sleep" tool. This tool helps AI to wait for a condition to be met before performing any other actions. We needed this tool because seems like AI does not have any perception of time so we implemented this to be more reliable with waiting. 


Why we need both of these tools in the same PR? 

Because sometimes we might need to wait for the Email to be received from the client.